object/option_value_list.go

```diff
@@ -44,6 +44,114 @@ func OptionValueListFromMap[T any](in map[string]T) OptionValueList {
 	return out
 }
 
+// IsTrue returns true if the specified key exists with an empty value or value
+// equal to 1, "1", "on", "t", true, "true", "y", or "yes".
+// All string comparisons are case-insensitive.
+func (ov OptionValueList) IsTrue(key string) bool {
+	val, ok := ov.Get(key)
+	if !ok {
+		return false
+	}
+
+	switch tval := val.(type) {
+	case string:
+		switch tval {
+		case "", "1",
+			"on", "ON", "On", "oN",
```

Comment:
Because when I started I thought there were a small enough number to mitigate the need for more computational power for case-insensitive compare. If you think it's better, I'll switch it.

Reply:
If we were crunching a ton of data sure, but the use-case here is an occasional compare of a few chars, so I'd opt for readability / less code. Then again, the set of possible values is static (unlikely to change) and works the same. So I am ok with it either way.